HybridCamera/src/ui/button/record/RecordButton+Event.swift

```diff
@@ -30,13 +31,24 @@ extension RecordButton: UIGestureRecognizerDelegate {
    /**
     * Long press
     */
-   @objc open func handleLongPress(sender: UITapGestureRecognizer) {
+   @objc open func handleLongPress(sender: UILongPressGestureRecognizer) {
       if [.ended, .cancelled, .failed].contains(sender.state) {/*long tap release*/
          handleLongPressRelease()
-      } else if case .began = sender.state {/*long tap just began*/
+      } else if sender.state == .began {/*long tap just began*/
+         startingPoint = sender.location(in: sender.view)
          handleLongPressBegan()
+      } else if sender.state == .changed {
+         let newPoint = sender.location(in: sender.view)
+         let additionalZoom = distance(from: startingPoint, to: newPoint) / 60 - 1
```

Comment:
We can get rid of the distance util method by doing: `hypot(startingPoint.x - newPoint.x, startingPoint.y - newPoint.y) / 60 -1` Why not use util method? Since hypot already works pretty great. and we don't do a lot of dist calls. If we had more use cases for dist I would put the dist method in extension CGPoint {func dist...} in a CGPointSugar.swift file in a src/common/ folder.

Reply:
Yeah, I thought about putting it in a new folder. But I really couldn't see where else it would be required...

Reply:
👍 Exactly. If stuff is only used once. its often smart to avoid creating another method. You want to keep method count low. to reduce Cyclomatic complexity which gives you higher code beat GPA score.

Reply:
Ill try your work on my iPhone tomorrow. See if it works 👍

Reply:
Did you test it already? Unfortunately, I couldn't code as much as the previous days today due to studying (as I told you a while ago). Gonna check out the code related to flipping the camera now 😃

Reply:
Yeh. focus on your exam man. Why do you have a test in the middle of the summer tho?

Reply:
Well, med school isn't easy. Unfortunately, you can't study anatomy like math or chemistry by understanding and reasoning :/